src/lib/zf/doxygen/040_using.dox

```diff
@@ -869,22 +870,51 @@ enp4s0f0/0f0         id=10    pid=8845
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # zf_stackdump dump
 ============================================================
-name=enp4s0f0/0f0
-  pool: pkt_bufs_n=17536 free=17025
-  config: tcp_timewait_ticks=666 tcp_finwait_ticks=666
+onload version= onload-8.1.3.40
+tcpdirect version=8.1.3 onload-8.1 tcpdirect-8.1 
+name=ens1f0/10a interface=ens1f0 vlan_id=65535
+  pool: pkt_bufs_n=17536 free=17024
+  config: tcp_timewait_ticks=666 tcp_finwait_ticks=666 ctpio_threshold=-1
   config: tcp_initial_cwnd=0 ms_per_tcp_tick=90
   alts: n_alts=0
-  stats: ring_refill_nomem=0 discard_csum_bad=0 discard_mcast_mismatch=0
+  stats: ring_refill_nomem=0 cplane_alien_ifindex=0
+         tcp_retransmits=0
+  discards: discard_csum_bad=0 discard_mcast_mismatch=0
          discard_crc_bad=0 discard_trunc=0 discard_rights=0
          discard_ev_error=0 discard_other=0 discard_inner_csum_bad=0
-         cplane_alien_ifindex=0
-nic0: vi=240 flags=0 intf=enp4s0f0 index=6 hw=1A1
-  txq: pio_buf_size=2048
+         non_tcpudp=0
+  hwport_to_nicno: [0,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,]
+nic0: vi=266 vi_flags=1000000 nic_flags=1 intf=ens1f0 index=14 hw=1C1
+  txq: pio_buf_size=4096 added=622350 removed=622349
 ============================================================
-UDP RX enp4s0f0/0f0:0
-  filter: lcl=172.16.130.252:8012 rmt=0.0.0.0:0
-  rx: unread=1 begin=0 process=0 end=1
-  udp rx: release_n=1 q_drops=0
+UDP TX ens1f0/10a:0 lcl=192.168.130.233:9001 rmt=192.168.130.232:9001 ttl=64
+  path: dst=192.168.130.232 src=192.168.130.233 nic=0
```

Comment:
Do we mind that this example has changed from RX to TX?

Reply:
Change was made to likely highlight the new TTL setting which I see is now missing from documentation changes.

Reply:
On line 974 you should add the addition of the TTL attribute to the UDP TX part of the stackdump output.